docker-cloud/cloud-swarm/connect-to-swarm.md

```diff
@@ -99,6 +99,48 @@ To switch back to the deployed swarm, re-run the `export DOCKER_HOST` command wi
 
 To learn more, see [Unset environment variables in the current shell](/machine/get-started/#unset-environment-variables-in-the-current-shell).
 
+## Use Docker for Mac and Windows (Edge) to connect to swarms
+
+On Docker for Mac and Docker for Windows current Edge releases,
+you can access your Docker Cloud account and connect directly to your swarms through their menus.
```

Comment:
Maybe s/through their menus/from the application? or from the UI? Since you talk about the UI below.